Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -445,42 +442,43 @@ def getJobOutput(self, jobID, _localDir=None):
     # same machine
     # workingDirectory = self.ceParameters.get( 'WorkingDirectory', DEFAULT_WORKINGDIRECTORY )
 
-    if not self.useLocalSchedd:
-      iwd = None
+    iwd = None
 
-      # TOREMOVE: once v7r0 will mainly be used, remove the following block that was only useful
-      # when path to output was not deterministic
-      status, stdout_q = commands.getstatusoutput('condor_q %s %s -af SUBMIT_Iwd' % (self.remoteScheddOptions,
+    # TOREMOVE: once v7r0 will mainly be used, remove the following block that was only useful
+    # when path to output was not deterministic
+    status, stdout_q = commands.getstatusoutput('condor_q %s %s -af SUBMIT_Iwd' % (self.remoteScheddOptions,
                                                                                      condorID))
-      self.log.verbose('condor_q:', stdout_q)
-      if status == 0 and self.workingDirectory in stdout_q:
-        iwd = stdout_q
-        pathToResult = iwd
-
-      # Use the path extracted from the pilotID
-      if iwd is None:
-        iwd = os.path.join(self.workingDirectory, pathToResult)
-
-      try:
-        mkDir(iwd)
-      except OSError as e:
-        errorMessage = "Failed to create the pilot output directory"
-        self.log.exception(errorMessage, iwd)
-        return S_ERROR(e.errno, '%s (%s)' % (errorMessage, iwd))
-
-      cmd = ['condor_transfer_data', '-pool', '%s:9619' % self.ceName, '-name', self.ceName, condorID]
-      result = executeGridCommand(self.proxy, cmd, self.gridEnv)
-      self.log.verbose(result)
-
-      errorMessage = "Failed to get job output from htcondor"
-      if not result['OK']:
-        self.log.error(errorMessage, result['Message'])
-        return result
-      # Even if result is OK, the actual exit code of cmd can still be an error
-      if result['OK'] and result['Value'][0] != 0:
-        varMessage = result['Value'][1].strip()
-        self.log.error(errorMessage, varMessage)
-        return S_ERROR('%s: %s' % (errorMessage, varMessage))
+    self.log.verbose('condor_q:', stdout_q)
+    if status == 0 and self.workingDirectory in stdout_q:
+      iwd = stdout_q
+      pathToResult = iwd
+
+    # Use the path extracted from the pilotID
+    if iwd is None:
+      iwd = os.path.join(self.workingDirectory, pathToResult)
+
+    try:
+      mkDir(iwd)
+    except OSError as e:
+      errorMessage = "Failed to create the pilot output directory"
+      self.log.exception(errorMessage, iwd)
+      return S_ERROR(e.errno, '%s (%s)' % (errorMessage, iwd))
+
+    cmd = ['condor_transfer_data', '-pool', '%s:9619' % self.ceName, '-name', self.ceName, condorID]
```

Comment:
I guess this needs testing with the local schedd

Reply:
Any advice?
I tested with the condor available on `lxplus` (so without DIRAC), I guess it's a local scheduler, isn't it?

I modified the code, if we use a local scheduler then the command should be:
``` python
 cmd = ['condor_transfer_data', condorID]
```